packages/liveblocks-react-ui/src/primitives/AiMessage/types.ts

```diff
@@ -0,0 +1,45 @@
+import type {
+  AiChatMessage,
+  AiReasoningPart,
+  AiTextPart,
+  AiUploadedImagePart,
+} from "@liveblocks/core";
+import type { ComponentType } from "react";
+
+import type { ComponentPropsWithSlot } from "../../types";
+
+export interface AiMessageContentComponents {
+  /**
+   * The component used to display text parts.
+   */
+  TextPart: ComponentType<{ part: AiTextPart }>;
+
+  /**
+   * The component used to display uploaded image parts.
+   */
+  UploadedImagePart: ComponentType<{ part: AiUploadedImagePart }>;
```

Comment:
@nimeshnayaju Should we remove all "UploadedImage" stuff before the launch? I'd prefer to add it back in after the launch once we are ready to support it?

Reply:
Yes! I'd like these removed before the launch. We still have few unknowns on attachments on both client and server side.